client/me/notification-settings/index.jsx

```diff
@@ -49,7 +49,7 @@ const NotificationSettings = React.createClass( {
 		}
 
 		if ( state.status === 'success' ) {
-			this.props.successNotice( this.translate( 'Settings saved successfully!' ), { duration: 3000 } );
+			this.props.successNotice( this.translate( 'Settings saved successfully!' ) );
```

Comment:
I guess we're relying upon route change to clear the notice instead of time now, but that's a functional change that I think should at least be mentioned/discussed. FWIW I see no problem with it though.